data-plane/pom.xml

```diff
@@ -42,6 +42,7 @@
 
     <!-- dependencies version -->
     <vertx.version>4.2.5</vertx.version>
```

Comment:
Any reason to leverage the rest of that on a different version?

Reply:
to be able to use Kafka client 3.2

Reply:
Ok, I guess aligned is _nice_, but no stopper, I guess